exit code != 0 when rolling back ?

I use deploy-rs in a gitlab pipeline and I would like the job to fail when the deployment fails.
It seems like deploy-rs still returns 0 upon fallback ?:
```
deploy-rs`
Updating GRUB 2 menu...
stopping the following units: audit.service, datadog-agent.service, docker.service, docker.socket, kmod-static-nodes.service, logrotate-checkconf.service, mount-pstore.service, network-local-commands.service, network-setup.service, nscd.service, resolvconf.service, systemd-modules-load.service, systemd-sysctl.service, systemd-timesyncd.service, systemd-udevd-control.socket, systemd-udevd-kernel.socket, systemd-udevd.service, systemd-vconsole-setup.service
👀 ❌ [wait] [ERROR] Error waiting for activation: Timeout elapsed for confirmation
🚀 ❌ [deploy] [ERROR] Waiting over SSH resulted in a bad exit code: Some(1)
🚀 ℹ️ [deploy] [INFO] Revoking previous deploys
Cleaning up project directory and file based variables 00:01
Job succeeded
```

I wonder if I can toggle this behavior via a flag ?

--- a/src/cli.rs
+++ b/src/cli.rs
@@ -388,6 +388,8 @@ pub enum RunDeployError {
     PromptDeployment(#[from] PromptDeploymentError),
     #[error("Failed to revoke profile: {0}")]
     RevokeProfile(#[from] deploy::deploy::RevokeProfileError),
+    #[error("Deployment failed, rolled back to previous generation")]
+    Rollback
 }
 
 type ToDeploy<'a> = Vec<(
@@ -577,7 +579,7 @@ async fn run_deploy(
                     }
                 }
             }
-            break;
+            return Err(RunDeployError::Rollback);
         }
         succeeded.push((deploy_data, deploy_defs))
     }

